### PR TITLE
Some trivial builderDefs->mkDerivation translations

### PR DESCRIPTION
**Pull request by joachifm**

### CODE REVIEW

File: pkgs/applications/misc/vue/default.nix [hunk 1]
```diff
@@ -1,50 +1,25 @@
...
-    baseName="vue";
-    version="3.2.2";
-    name="${baseName}-${version}";
-    url="releases.atech.tufts.edu/jenkins/job/VUE/64/deployedArtifacts/download/artifact.2";
-    hash="0sb1kgan8fvph2cqfxk3906cwx5wy83zni2vlz4zzi6yg4zvfxld";
+stdenv.mkDerivation rec {
+  name = "vue-${version}";
+  version = "3.2.2";
+  src = fetchurl {
+    url = "releases.atech.tufts.edu/jenkins/job/VUE/64/deployedArtifacts/download/artifact.2";
```
Comment (spwhitt) [inline pkgs/applications/misc/vue/default.nix]: Encountering the following error on build:

```
unpacking source archive /nix/store/l357ff5b71vzmxbnapf7xgq44nhcn129-artifact.2
do not know how to unpack source archive /nix/store/l357ff5b71vzmxbnapf7xgq44nhcn129-artifact.2
```

**Comment by domenkozar** [comment]
Awesome! Once you fix the error(s) I'll merge

File: pkgs/applications/misc/vue/default.nix (hunk 1 above)
Comment (joachifm) [inline pkgs/applications/misc/vue/default.nix]: Ach, that is vexing, I skipped trying to build vue myself as its just a 1:1 translation ... Will look into this and push a fixed version shortly. Thanks

Comment (joachifm) [inline pkgs/applications/misc/vue/default.nix]: Obviously, vue doesn't need unpacking ... fixed by skipping that step.

**Comment by joachifm** [comment]
Fixed vue. All these packages build for me now.

**Pull request merged**